frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1561,20 +1563,25 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
         statusCheck: {
             __default: null as WebAnalyticsStatusCheck | null,
             loadStatusCheck: async (): Promise<WebAnalyticsStatusCheck> => {
-                const [pageviewResult, pageleaveResult, pageleaveScroll] = await Promise.allSettled([
-                    api.eventDefinitions.list({
-                        event_type: EventDefinitionType.Event,
-                        search: '$pageview',
-                    }),
-                    api.eventDefinitions.list({
-                        event_type: EventDefinitionType.Event,
-                        search: '$pageleave',
-                    }),
-                    api.propertyDefinitions.list({
-                        event_names: ['$pageleave'],
-                        properties: ['$prev_pageview_max_content_percentage'],
-                    }),
-                ])
+                const [pageviewResult, pageleaveResult, pageleaveScroll, pageViewsWithoutSessionsResult] =
+                    await Promise.allSettled([
+                        api.eventDefinitions.list({
+                            event_type: EventDefinitionType.Event,
+                            search: '$pageview',
+                        }),
+                        api.eventDefinitions.list({
+                            event_type: EventDefinitionType.Event,
+                            search: '$pageleave',
+                        }),
+                        api.propertyDefinitions.list({
+                            event_names: ['$pageleave'],
+                            properties: ['$prev_pageview_max_content_percentage'],
+                        }),
+                        api.query<HogQLQuery>({
+                            kind: NodeKind.HogQLQuery,
+                            query: "SELECT events.properties.$lib as lib FROM events WHERE event='$pageview' AND or(events.$session_id IS NULL, events.$session_id = '') AND timestamp > today() LIMIT 1",
+                        }),
```

Comment:
Does it make sense to always use `today` rather than using `logic.dateFilter`?

Reply:
I chose today rather than the whole date period because
* I don't want to slow down the page too much by querying too much data
* I'd like the error message to disappear when the error is fixed, within a reasonable amount of time